commit e7d83ebe22feb37a728aeb3c2281cf85a4b4c092
Author: Simon van der Veldt <simon.vanderveldt@gmail.com>
Date:   Tue Feb 6 15:36:25 2018 +0100

    Add missing SND_PISOUND selects dependency to SND_RAWMIDI
    
    Without it the Pisound module fails to compile.
    See https://github.com/raspberrypi/linux/issues/2366
    
    Signed-off-by: Simon van der Veldt <simon.vanderveldt@gmail.com>

diff --git a/sound/soc/bcm/Kconfig b/sound/soc/bcm/Kconfig
index bf754c0b9fc1..212bbaa6f1ca 100644
--- a/sound/soc/bcm/Kconfig
+++ b/sound/soc/bcm/Kconfig
@@ -192,5 +192,6 @@ config SND_BCM2708_SOC_ALLO_DIGIONE
 config SND_PISOUND
 	tristate "Support for Blokas Labs pisound"
 	depends on SND_BCM2708_SOC_I2S || SND_BCM2835_SOC_I2S
+	select SND_RAWMIDI
 	help
 	  Say Y or M if you want to add support for Blokas Labs pisound.
